### PR TITLE
PS5_script

### PR DESCRIPTION
**Pull request by casey-martin**

### CODE REVIEW

**Comment by casey-martin** [comment]
Rpubs HTML: http://rpubs.com/casey_martin/277301

Markdown answers are hardcoded, rather than invoking the python script via {bash}. If this is egregious enough, I can resubmit.

File: problem_1.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from collections import Counter
+
+myfile = open("/home/dalgarno/Documents/GAW/data-sets/bed/lamina.bed", 'r')
+mybed = [line.split('\t') for line in myfile]
+
+mychroms = [i[0] for i in mybed[2:]]
+
+mychroms = (Counter(mychroms))
+mymaxkey = (max(mychroms.keys(), key=(lambda key: mychroms[key])))
```
Comment (kriemo) [inline problem_1.py]: nice use of lambda functions and list comprehensions 👍

File: problem_2.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from Bio import SeqIO
+
+myfile = '/home/dalgarno/Documents/GAW/data-sets/fastq/SP1.fq'
+
+myfastq = SeqIO.parse(open(myfile), 'fastq')
```
Comment (kriemo) [inline problem_2.py]: nice reuse of an existing parser. If you end up writing code to parse fastqs with 10s of millions of records,  I would suggest using a different parser or using the strategies discussed in this post (https://www.biostars.org/p/10353/).  The SeqIO parser can be pretty slow.

Comment (casey-martin) [inline problem_2.py]: I'll check these out. I'm about to start on some larger datasets, so these should come in handy. Thanks!

**Pull request closed without merging**